docs/docs/creating-a-source-plugin.md

```diff
@@ -142,7 +142,7 @@ In order to create a parent/child relationship, when calling `createNode` for th
 
 _Examples_
 
-[Here's the above example](https://github.com/gatsbyjs/gatsby/blob/master/packages/gatsby-transformer-remark/src/on-node-create.js#L34-L64) from the `gatsby-transformer-remark` source plugin.
+[Here's the above example](https://github.com/gatsbyjs/gatsby/blob/master/packages/gatsby-transformer-remark/src/on-node-create.js#L39-L67) from the `gatsby-transformer-remark` source plugin.
```

Comment:
Maybe we should pin link to commit SHA so those lines don't move around?
```suggestion
[Here's the above example](https://github.com/gatsbyjs/gatsby/blob/72077527b4acd3f2109ed5a2fcb780cddefee35a/packages/gatsby-transformer-remark/src/on-node-create.js#L39-L67) from the `gatsby-transformer-remark` source plugin.
```

Reply:
Yes. Thats better. 

PS: I checked the other links with hash and they are not changed since then